src/main/java/CarRentalService/CRS/Controllers/BookingController.java

```diff
@@ -0,0 +1,97 @@
+package CarRentalService.CRS.Controllers;
+
+import CarRentalService.CRS.Models.Booking;
+import CarRentalService.CRS.Services.BookingService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/booking")
+public class BookingController {
+
+    @Autowired
+    private BookingService bookingService;
+
+    @GetMapping("")
```

Comment:
You don't need this `("")`, just `@GetMapping`

Reply:
Fixed